src/Components/Patient/PatientRegister.tsx

```diff
@@ -1035,17 +1036,17 @@ export const PatientRegister = (props: PatientRegisterProps) => {
           ) : (
             <>
               <>
+                <ButtonV2
+                  className="mb-8 mx-4"
+                  onClick={(_) => {
+                    setShowImport(true);
+                    setQuery({ extId: "" }, { replace: true });
+                  }}
+                >
+                  {" "}
+                  Import From External Results
+                </ButtonV2>
```

Comment:
How about also adding an import icon?

```suggestion
                <ButtonV2
                  className="mb-8 mx-4 flex gap-2 items-center"
                  onClick={(_) => {
                    setShowImport(true);
                    setQuery({ extId: "" }, { replace: true });
                  }}
                >
                  <CareIcon className="care-l-import text-lg" />
                  Import From External Results
                </ButtonV2>
```

Reply:
Looks good! Nice one.